examples/shell/shell_common/cmd_server.cpp

```diff
@@ -172,8 +172,10 @@ static CHIP_ERROR CmdAppServerClusters(int argc, char ** argv)
 
     for (int i = 0; i < emberAfEndpointCount(); i++)
     {
-        EndpointId endpoint   = emberAfEndpointFromIndex(i);
-        uint16_t clusterCount = emberAfClusterCount(endpoint, server);
+        EndpointId endpoint = emberAfEndpointFromIndex(i);
+
+        VerifyOrReturnError(emberAfClusterCount(endpoint, server) <= UINT8_MAX, CHIP_ERROR_INCORRECT_STATE);
+        uint8_t clusterCount = static_cast<uint8_t>(emberAfClusterCount(endpoint, server));
```

Comment:
Why?  `emberAfClusterCount` returns `uint8_t`.

Reply:
Probably mistake after several updates. Reverted to just store as uint8_t.